exercises/connect/cases_test.go

```diff
@@ -0,0 +1,96 @@
+package connect
+
+// Source: exercism/x-common
+// Commit: 6c6a395 improving description again
+
+const testVersion = 2
```

Comment:
we would want the `testVersion` to go in `example.go` so that submitted solutions have to show their `testVersion` (allowing reviewers of the code to easily see what version of the tests a solution was coded against). Putting it in `cases_test.go` would mean that nobody has to declare it